### PR TITLE
BUGFIX: Translate FlashMessage if asset is still in use in media browser inspector 

### PR DESCRIPTION
**Pull request by KommunikativCh**
Bugfix for #5085 no error if asset still in use and show flashmessage solution set class neos-notification-container as id

In media browser inspector if assets are deleted wich are still used no FlashMessage was showed  and on second delete an error pages was showed.

Solution:
In Neos.Media:Browser/Resources/Private/Layouts/Default.html change class neos-notification-container to id neos-notification-container, because Notification.js / Toast is searching for id neos-notification-container to insert FlashMessages.

`       <div class="neos-media-content{f:if(condition: '{tags -> f:count()} > 25', then: ' neos-media-aside-condensed')}">
            <div class="neos-media-assets">
                <div **id="neos-notification-container"**>
                    <f:render partial="FlashMessages"/>
                </div>
                <f:render section="Content"/>
            </div>
            <aside class="neos-media-aside">`

related: #5085

**Checklist**

- [ ] Code follows the PSR-2 coding style
- [ ] Tests have been created, run and adjusted as needed
- [x] The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)
- [x] Reviewer - PR Title is brief but complete and starts with `FEATURE|TASK|BUGFIX`
- [ ] Reviewer - The first section explains the change briefly for change-logs
- [ ] Reviewer - Breaking Changes are marked with `!!!` and have upgrade-instructions

### CODE REVIEW

**Comment by KommunikativCh** [comment]
During testing it was noticed that the message text was always displayed in the default language. this has now been fixed with the [commit](https://github.com/neos/neos-development-collection/pull/5133/commits/bc3eaf049f97004dc9d4d632a08b483f19953215).

**Comment by dlubitz** [comment]
Thanks @KommunikativCh for this PR. But I couldn't reproduce the error in the issue at all (see [#5085](https://github.com/neos/neos-development-collection/issues/5085#issuecomment-2286215685)). 

As this still seems to be a valid point and the `initializeObject` get's overriden by the class, shall we merge it anyways? What do you think?

See also: https://3v4l.org/0dUtS

**Review by dlubitz — APPROVED** [review]
Ok, now I could reproduce your observation, with the untranslated error message! Makes sence now!

File: Neos.Media.Browser/Classes/Controller/AssetController.php [hunk 1]
```diff
@@ -70,8 +70,11 @@
  */
 class AssetController extends ActionController
 {
-    use CreateContentContextTrait;
     use BackendUserTranslationTrait;
+    use BackendUserTranslationTrait {
+        BackendUserTranslationTrait::initializeObject as BackendUserTranslationTraitInitializeObject;
```
Comment (bwaidelich) [inline Neos.Media.Browser/Classes/Controller/AssetController.php]: I'm not very familiar with method aliasing, but shouldn't this be lower-cased if you use it like that in line 176?
```suggestion
        BackendUserTranslationTrait::initializeObject as backendUserTranslationTraitInitializeObject;
```

Comment (bwaidelich) [reply Neos.Media.Browser/Classes/Controller/AssetController.php]: @KommunikativCh @dlubitz ping

Comment (dlubitz) [reply Neos.Media.Browser/Classes/Controller/AssetController.php]: I think we should align this. But in PHP methods and aliases are case-insensitve, so it works anyways.

Comment (dlubitz) [reply Neos.Media.Browser/Classes/Controller/AssetController.php]: I quickly changed it. @bwaidelich

Comment (bwaidelich) [reply Neos.Media.Browser/Classes/Controller/AssetController.php]: Thanks. I would have changed it myself, but wasn't sure if that was done by intention

**Review by bwaidelich — APPROVED** [review]
I don't like Traits. And I especially don't like Traits that need to be initialized. And I _espeically_ don't like Traits that use `initializeObject` to be initialized – But that all got nothing to do with your fix ofc.
So if that's the way it works +1 by reading and thanks for your contribution! :)

**Comment by dlubitz** [comment]
> I don't like Traits. And I especially don't like Traits that need to be initialized. And I _espeically_ don't like Traits that use `initializeObject` to be initialized – But that all got nothing to do with your fix ofc. So if that's the way it works +1 by reading and thanks for your contribution! :)

Totally agree. This case is a very good example, how traits shouldn't be used 🤷‍♂️

**Pull request merged**